youtube_dl/extractor/adn.py

```diff
@@ -31,30 +31,30 @@
 
 
 class ADNIE(InfoExtractor):
-    IE_DESC = 'Anime Digital Network'
-    _VALID_URL = r'https?://(?:www\.)?animedigitalnetwork\.fr/video/[^/]+/(?P<id>\d+)'
+    IE_DESC = 'Animation Digital Network'
+    _VALID_URL = r'https?://(?:www\.)?animationdigitalnetwork\.fr/video/[^/]+/(?P<id>\d+)'
```

Comment:
What my proposition 11days ago. But I'm not sure it's really useful since I doubt that anybody will use it and it may increase complexity a bit but I guess it may happens. ¯\_(ツ)_/¯